src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        truck.doWork();
+        truck.stopWork();
+        Bulldozer bulldozer = new Bulldozer();
+        bulldozer.doWork();
+        bulldozer.stopWork();
+        Excavator excavator = new Excavator();
```

Comment:
please be attentive with checklist https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop